docs/infrastructure/infra-ui.asciidoc

```diff
@@ -59,4 +59,10 @@ Use the time selector to focus on a specific timeframe.
 Set auto-refresh to keep up-to-date information coming in, or stop
 refreshing to focus on historical data without new distractions.
 
+[float]
+[[infra-configure-source]]
+=== Adapt to your metric source
```

Comment:
What about:
> If you're using a custom index pattern to store your metrics, or want to limit the entries presented in a space, the  source configuration dialog enables you to change the index pattern and other relevant settings.

Reply:
I was trying to stay "in style" with the way the other sections on this page are phrased. Do we want to deviate or rephrase them too?

Reply:
Can you elaborate on that? This is the only section I see that is phrased as a question. Because of that it felt a little out of place to me.

Reply:
The question was one way I figured I could state a condition while staying in the imperative mood used by most of the other sections.

I'm not familiar with the intent that the original author had when choosing that mood. I'd be happy to drop it if it's not something dictated by a docs style guide.

Reply:
Thanks for clarifying. I think maybe it's the "You are..." that's throwing me off. Would this work for you?
> Using a custom index pattern to store the metrics, or want to limit the entries presented in a space? Use configure source to change the index pattern and other settings.

Reply:
that sounds good, thanks :+1: